typings/index.test-d.ts

```diff
@@ -857,10 +857,10 @@ declare const categoryChannel: CategoryChannel;
   expectType<Promise<VoiceChannel>>(categoryChannel.createChannel('name', { type: 'GUILD_VOICE' }));
   expectType<Promise<TextChannel>>(categoryChannel.createChannel('name', { type: 'GUILD_TEXT' }));
   expectType<Promise<NewsChannel>>(categoryChannel.createChannel('name', { type: 'GUILD_NEWS' }));
-  expectDeprecated(categoryChannel.createChannel('name', { type: 'GUILD_STORE' }));
+  expectType<Promise<StoreChannel>>(categoryChannel.createChannel('name', { type: 'GUILD_STORE' }));
```

Comment:
This should still be expected to be deprecated, right?

Reply:
No because that check is present above along with other deprecation checks related to Store channels. I just changed that to make sure we're checking all types here